### PR TITLE
Add React and Angular to Readmes

### PR DESCRIPTION
**Pull request by tnorling**
- Adds links to Angular and React wrappers on main readme and browser readme
- Adds npm badges to React Readme

### CODE REVIEW

**Review by jo-arroyo — APPROVED** [review]
LGTM! Thanks for updating this!

**Review by pkanher617 — APPROVED** [review]

File: lib/msal-browser/README.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
 | --- | --- | --- |
 
 1. [About](#about)
-2. [FAQ](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/FAQ.md)
-3. [Releases](#releases)
-4. [Prerequisites](#prerequisites)
-5. [Installation](#installation)
-6. [Usage](#usage)
+1. [FAQ](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/FAQ.md)
+1. [Roadmap](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/roadmap.md)
```
Comment (sameerag) [inline lib/msal-browser/README.md:10]: Do we want to list releases per library somewhere?

**Review by sameerag — APPROVED** [review]
lgtm. Small comment.

File: lib/msal-browser/README.md (hunk 1 above)
Comment (tnorling) [reply lib/msal-browser/README.md:10]: Changelog and Github Releases are where this information would be found.

Comment (tnorling) [reply lib/msal-browser/README.md:10]: We could link to the Changelog here

Comment (tnorling) [reply lib/msal-browser/README.md:11]: Done

**Pull request merged**